grafana/provisioning/dashboards/dev/pubsub.json

```diff
@@ -1,1576 +1,1576 @@
 {
-    "annotations": {
-        "list": [
-            {
-                "builtIn": 1,
-                "datasource": "-- Grafana --",
-                "enable": true,
-                "hide": true,
-                "iconColor": "rgba(0, 211, 255, 1)",
-                "name": "Annotations & Alerts",
-                "type": "dashboard"
-            }
-        ]
+  "annotations": {
```

Comment:
spacing is different. i don't mind changing the spacing but let's do that is separate prs to make the diff easier to read.

would be even better to add prettier if you don't mind

Reply:
that was it!  i should have seen that - adjusted the spacing to match and now we see only actual differences.

Not sure if I should keep the change from 'Cloudwatch' to 'Cloudwatch-1'

Verified that the re-upload of the new (exported and spacing adjusted) dashboards does match what is presently on the site.

Also verified that the dashboards in dev match what was on the site prior to the gateway changes.

It seems that the change to cloudwatch-1 is not significant?